website/server/libs/slack.js

```diff
@@ -77,14 +89,75 @@ function sendFlagNotification ({
   });
 }
 
+function sendInboxFlagNotification ({
+  authorEmail,
+  flagger,
+  message,
+  userComment,
+}) {
+  if (SKIP_FLAG_METHODS) {
+    return;
+  }
+  let titleLink;
+  let authorName;
+  let title = `Flag in ${flagger.profile.name}'s Inbox`;
+  let text = `${flagger.profile.name} (${flagger.id}; language: ${flagger.preferences.language}) flagged a PM`;
+  let footer = `<${SLACK_FLAGGING_FOOTER_LINK}?userId=${flagger.id}&chatMessageId=${message.id}|Flag this message>`;
```

Comment:
This footer link isn't needed for private messages because mods won't be flagging them  (the only reason for a mod to flag a group message is to hide it from all non-mods and that doesn't apply to private messages - if the user wants to not see the message, they can delete it after flagging it).

footer can be set to an empty string.

Reply:
so I didn't had to update https://github.com/HabitRPG/flag-o-rama/pull/5 ?

Reply:
In my opinion, there's no need to update it. However since you've written that code, I will check with the other mods to see if there is any use case that I haven't considered. I feel though that letting the user have the final control over the visibility of their messages is the best option. I'll report back. :)